ansible/README.md

```diff
@@ -0,0 +1,173 @@
+# Collector Ansible Tools
+
+The files contained within this directory are [ansible](https://www.ansible.com/resources/get-started)
+roles and playbooks written to simplify VM life-cycle control, and execution of testing, both
+during development and on CI systems[^1].
+
+## Prerequisites
+
+In order to run these playbooks you must have a python version >=3.9.
+To install ansible and the necessary dependencies, simply run:
+
+```
+$ pip3 install -r requirements.txt
+```
+
+On a Mac, the default python3 is 3.7, so use brew to install the latest ansible:
+
+```
+$ brew install ansible
+$ pip3 install -r requirements.txt
+```
```

Comment:
Another option we could consider is to build a docker image that already has all of this pre-installed. Might be overkill but could also make it even easier to use. But if that is something we want, we can obviously do it in a follow up PR.